### PR TITLE
Add rolling update strategy to docs

### PR DESCRIPTION
**Pull request by reset**
Documentation for the Rolling Update Strategy

gif-keyboard-3856532294397689321

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/docs/run-packages-update-strategy.html.md [hunk 1]
```diff
@@ -4,17 +4,31 @@ title: Update Strategy
 
 # Update Strategy
 
-The Habitat supervisor can be configured to implement an optional _update strategy_, which describes how the supervisor and its peers within a service group should respond when a new version of a package is available.
+The Habitat supervisor can be configured to leverage an optional _update strategy_, which describes how the supervisor and its peers within a service group should respond when a new version of a package is available.
 
-To use an update strategy, the supervisor be configured to watch a [depot](/docs/concepts-depot) for new versions. Habitat provides a public depot, but you can also run one inside your own datacenter using the `core/hab-depot` package.
+To use an update strategy, the supervisor be configured to watch a [depot](/docs/concepts-depot) for new versions.
```
Comment (mwrock) [inline www/source/docs/run-packages-update-strategy.html.md]: should be configured

File: www/source/docs/run-packages-update-strategy.html.md [hunk 2]
```diff
@@ -4,17 +4,31 @@ title: Update Strategy
...
+Habitat supports two update strategies: `rolling` and `at-once`.
 
 To start a supervisor with the auto-update strategy, pass the `--strategy` argument to a supervisor start command, and optionally specify the depot URL:
 
-       hab start yourorigin/yourapp --strategy at-once --url https://willem.habitat.sh/v1/depot
+       hab start yourorigin/yourapp --strategy rolling --url https://willem.habitat.sh/v1/depot
+
+### Rolling Strategy
+
+This strategy ensures that all versions of a service within a collection of supervisor's service group is running the same version of software. An update leader is elected which all supervisors within a service group will update around. All update followers will first ensure they are running the same version of a service that their leader is, and then, the leader will poll a depot for a newer version of the service's package.
```
Comment (mwrock) [inline www/source/docs/run-packages-update-strategy.html.md]: should "collection of" be removed?

Comment (mwrock) [inline www/source/docs/run-packages-update-strategy.html.md]: "that their leader is running"

File: www/source/docs/run-packages-update-strategy.html.md [hunk 3]
```diff
@@ -4,17 +4,31 @@ title: Update Strategy
...
+
+Once the update leader finds a new version it will update and wait until all other alive members in the service group have also been updated before once again attempting to find a newer version of software to update to. Updates will happen more or less one at a time until completion with the exception of a new node being introduced into the service group during the middle of an update.
+
+If your service group is also running with the `--topology leader` flag, the leader of that election will never become the update leader, so all followers within a leader topology will update first.
+
+It's important to note that because we must perform a leader election to determine an update leader, *you need to have at least 3 supervisors running a service group to take advantage of the rolling update strategy*.
+
+### At-Once Strategy
+
+This strategy does no peer coordination with other supervisors in the service group; it merely updates the underlying Habitat package whenever it detects that a new version has been published to a depot. Each supervisor will poll
```
Comment (mwrock) [inline www/source/docs/run-packages-update-strategy.html.md]: "Each supervisor will poll" looks like an incomplete sentance.

**Comment by mwrock** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/186442111).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/186442111).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**